ui/ducks/domains.test.ts

```diff
@@ -0,0 +1,238 @@
+import { fetchResolutions, shouldSkipEnsResolutionSnap } from './domains';
+// eslint-disable-next-line import/order
+import * as storeActions from '../store/actions';
+
+jest.mock('../store/actions', () => ({
+  ...jest.requireActual('../store/actions'),
+  handleSnapRequest: jest.fn(() => Promise.resolve({ resolvedAddresses: [] })),
+}));
+const mockedStoreActions = jest.mocked(storeActions);
+
+const preinstalledSnapState = {
+  snaps: {
+    'npm:@metamask/ens-resolver-snap': {
+      blocked: false,
+      enabled: true,
+      id: 'npm:@metamask/ens-resolver-snap',
+      initialPermissions: {
+        'endowment:ethereum-provider': {},
+        'endowment:name-lookup': {},
+        'endowment:network-access': {},
+      },
+      localizationFiles: [],
+      manifest: {
+        description: 'A Snap used for ENS name resolution',
+        initialPermissions: {
+          'endowment:ethereum-provider': {},
+          'endowment:name-lookup': {},
+          'endowment:network-access': {},
+        },
+        manifestVersion: '0.1',
+        proposedName: 'Ethereum Name Service resolver',
+        repository: {
+          type: 'git',
+          url: 'https://github.com/MetaMask/ens-resolver-snap.git',
+        },
+        source: {
+          location: {
+            npm: {
+              filePath: 'dist/bundle.js',
+              iconPath: 'images/icon.svg',
+              packageName: '@metamask/ens-resolver-snap',
+              registry: 'https://registry.npmjs.org/',
+            },
+          },
+          shasum: 'BizRmzfV+oKEIlvph12McsIqzzDECIw/Td7Lx+/cios=',
+        },
+        version: '0.1.2',
+      },
+      preinstalled: true,
+      removable: false,
+      status: 'crashed',
+      version: '0.1.2',
+      versionHistory: [
+        { date: 1726481055390, origin: 'metamask', version: '0.1.2' },
+      ],
+    },
+  },
+  subjects: {
+    'npm:@metamask/ens-resolver-snap': {
+      origin: 'npm:@metamask/ens-resolver-snap',
+      permissions: {
+        'endowment:ethereum-provider': {
+          caveats: null,
+          date: 1726481055390,
+          id: '5BFL-61JzLdm9GD-fxBy2',
+          invoker: 'npm:@metamask/ens-resolver-snap',
+          parentCapability: 'endowment:ethereum-provider',
+        },
+        'endowment:name-lookup': {
+          caveats: null,
+          date: 1726481055390,
+          id: 'b0q0p5kbA95LB1pRB9lsm',
+          invoker: 'npm:@metamask/ens-resolver-snap',
+          parentCapability: 'endowment:name-lookup',
+        },
+        'endowment:network-access': {
+          caveats: null,
+          date: 1726481055390,
+          id: 'XMk2mHrkgrRgXC8kjwtzV',
+          invoker: 'npm:@metamask/ens-resolver-snap',
+          parentCapability: 'endowment:network-access',
+        },
+      },
+    },
+  },
+};
+
+describe('domain resolution using preinstalled snap', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should not skip snap for mainnet', async () => {
```

Comment:
Minor: Our [contributor docs](https://github.com/MetaMask/contributor-docs/blob/main/docs/unit-testing.md#use-it-to-specify-the-desired-behavior-for-the-code-under-test) say to not use `should` in our test descriptions.

However, a quick global search in the repo shows that we have over three thousand tests breaking this rule.

![image](https://github.com/user-attachments/assets/56ea67ed-b661-4608-bded-7932bab71fba)

It's possible that the guidelines were created after these tests were written. 

I'll leave it up to you to change or leave as is! I thought I'd mention it because I received the same comment on a recent PR 😄